buildrunner/docker/builder.py

```diff
@@ -52,9 +128,23 @@ def __init__(
             timeout=timeout,
         )
         self.docker_registry = docker_registry
-        self.image = None
+        self._image = None
         self.intermediate_containers = []
 
+    @property
+    def image(self):
+        """
+        Get the image ID of the built image.
+        """
+        return self._image
+
+    @image.setter
+    def image(self, value):
+        """
+        Set the image ID of the built image.
+        """
+        self._image = value
```

Comment:
Why the indirection if we aren't doing any magic in the getter/setter?

Reply:
I wasn't able to mock the image property without using @property and the setter. I have since refactored the legacy build code to be in a single function and I don't need to mock the image property any more. So I removed this. I am glad you caught this, since I never liked having it in here.